### PR TITLE
Color block switching

### PR DESCRIPTION
**Pull request by palmfjord**
Enables to switch colors for all blocks, out of the box 📦

screenshot 2018-11-19 at 14 18 41
screenshot 2018-11-19 at 14 18 52

### CODE REVIEW

File: src/components/blockHelpers.tsx [hunk 1]
```diff
@@ -1,8 +1,48 @@
+import { colors } from '@hedviginsurance/brand'
 import styled from 'react-emotion'
+import { colorComponentColors } from '../blocks/BaseBlockProps'
 
-export const SectionWrapper = styled('section')({
-  padding: '7rem 0',
-})
+const TMP_OFF_BLACK_DARK = '#141033' // TODO move this to brand package
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/components/blockHelpers.tsx]: 🤷‍♀️ they're in the figma file from E&W

File: src/components/blockHelpers.tsx [hunk 2]
```diff
@@ -1,8 +1,48 @@
...
+    },
+    green: {
+      color: colors.WHITE,
+      background: colors.GREEN,
+    },
+    purple: {
+      color: colors.WHITE,
+      background: colors.PURPLE,
+    },
+  } as any)[color || 'standard'])
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/components/blockHelpers.tsx]: i know :l i just don't know how to type this

[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/components/blockHelpers.tsx]: Is it complaining about a lack of index signature, or what's the issue? 🙂

File: src/components/blockHelpers.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/components/blockHelpers.tsx]: I guess they are a thing then! Lets get it into brand though 😉

File: src/components/blockHelpers.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/components/blockHelpers.tsx]: yeh true, coming back to this a few moments later idk what can be so hard 😂

File: src/components/blockHelpers.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/components/blockHelpers.tsx]: https://github.com/HedvigInsurance/brand/pull/8 ✅

**Pull request merged**